README.md

```diff
@@ -172,6 +178,131 @@ The OSIsoft connector now has the ability to support OMF. To enable OMF follow t
+   ```
+   {
+   "ServerConfig":{
+      "IP":"192.168.1.8",
+      "WebID":"F1DS4knnwtuIDUC1RLy6XJGV4QREVTS1RPUC0wQ1MyUDFT",
+      "Credentials":"dsZtOnBfdsjklfh==",
+      "namespace":"testNameSpace1",
+      "tenantId":"4da89f7d-b3ce-48a6-b906-b96c367000a9"
+   },
+   "eWONConfig":{
+      "CertificatePath":"/usr"
+   },
+   "AppConfig": {
+       "LoggingLevel": 4,
+       "CycleTimeMs": 1000,
+       "CommunicationType": "omfOcs",
+       "PostDuplicateTagValues": false
+       }
+   }
+   ```
+   1. Upload the updated ConnectorConfig.json file to the /usr directory of your Flexy.
+
+### Flexy OCS BASIC Script
+1. Create a tag in your Flexy for the Basic script to store information.
+   1. On the Flexy's web page, click to expand the "Tags" section.
+   1. Click the "Values" tab.
+   1. Click the "MODE" slider to say "setup"
+   1. Click the "add" button.
+   1. Enter the tag name "tokenReq"
+   1. Under the "I/O Server Setup" section, change the type to "Integer"
+   1. Leave everything else as the default values and click "Add Tag" at the bottom right of the window.
+
+1. Obtain URL encoded client Id and client secret
+   1. In a web browser such as Internet Explorer or Chrome, navigate to the website https://www.urlencoder.org/
+   1. Copy your client ID into the encoder section of that website
+   1. Leave the destination character set as UTF-8
+   1. Leave the destination newline seperator as LF (Unix)
+   1. Hit Encode
+   1. Copy out the encoded client id and save for later use.
+   1. Copy your client secret into the encoder section of that website
+   1. Leave the destination character set as UTF-8
+   1. Leave the destination newline seperator as LF (Unix)
+   1. Hit Encode
+   1. Copy out your encoded client secret for later use.
+
+
+1. Copy the below basic script to the Init Section of the Flexy Basic IDE
+   1. On the Flexy's web page, click to expand the "Setup" section.
+   1. Click on the "BASIC IDE" tab.
+   1. Copy and paste the below basic script into the Init section
+   1. Change the xxxxxx for client ID to be your URL encoded client id that you saved previously.
+   1. Change the yyyyyy for client secret your URL encoded client secret you saved previously.
+   1. Click on "File"
+   1. Click on "Save"
+   1. Click on the script execution slider that says "Stopped" to switch it to say "Running"
+   
+```
+// Timer 60 seconds * 30 minutes
+TSET 1, (60*30)
+ONTIMER 1, "@updateToken()"
+END
+
+Function updateToken()
+
+  URL$ = "https://dat-b.osisoft.com/identity/connect/token"
+  Header$ = ""
+  ClientId$ = "xxxxxx"
+  ClientSecret$ = "yyyyyy"
+  DataToSend$ = "client_id="+ ClientId$ +"&client_secret="+ ClientSecret$+"&grant_type=client_credentials"
+  File$ = "/usr/token.json"
+
+  REQUESTHTTPX URL$, "POST",Header$ , DataToSend$, "", File$
+
+  // change tag value here to mark as set
+  tokenReq@ = (tokenReq@ + 1)
+ENDFN
+```
```

Comment:
Shouldnt this be a file added to the release zip?

Reply:
It's pretty short were copy and paste is just as easy as the import. I suppose either way would be fine. Should i move this and suggest they import it into the basic ide?

Reply:
I think its a lot safer and to have them import it from a file. That way you eliminate the chance for copy/paste errors.

Reply:
This chunk of code also needs to be formatted, a lot of inconstant spacing.